src/main/java/ru/yandex/practicum/filmorate/services/FilmService.java

```diff
@@ -0,0 +1,69 @@
+package ru.yandex.practicum.filmorate.services;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+import ru.yandex.practicum.filmorate.exceptions.FilmException;
+import ru.yandex.practicum.filmorate.model.Film;
+import ru.yandex.practicum.filmorate.storages.InMemoryFilmStorage;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Service
+@RequiredArgsConstructor
+@Slf4j
+public class FilmService {
+    InMemoryFilmStorage inMemoryFilmStorage = new InMemoryFilmStorage();
```

Comment:
Лучше обозначать интерфейс,  а создавать объект уже с определенной реализацией:
```suggestion
    FilmStorage inMemoryFilmStorage = new InMemoryFilmStorage();
```
Нужно будет добавить метод `getFilms` в интерфейс